src/components/DocumentationTopic.vue

```diff
@@ -369,7 +369,9 @@ export default {
     };
   },
   computed: {
-    normalizedSwiftPath: ({ normalizePath, swiftPath }) => (normalizePath(swiftPath)),
+    normalizedSwiftPath: ({ normalizePath, swiftPath }) => (
```

Comment:
Can we add/update a unit test for this? Maybe the `ViewMore` related specs or something since that seems to be the thing using this computed property?

Reply:
😬